components/Shipping/basic_usage.rst

```diff
@@ -0,0 +1,656 @@
+Basic Usage
+===========
+
+Shipping Category
+-----------------
+
+Every shipping category has two identifiers, an ID and name. You can access those by calling ``->getId()`` and ``->getName()``
+methods respectively. The name is mutable, so you can change it by calling ``->setName('Regular')`` on the shipping category instance.
+
+Shipping Method
+---------------
+
+Every shipping method has two identifiers, an ID and name. You can access those by calling ``->getId()`` and ``->getName()``
+methods respectively. The name is mutable, so you can change it by calling ``->setName('FedEx')`` on the shipping method instance.
+
+Setting Shipping Category
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Every shipping method can have shipping category. You can simply set or unset it by calling ``->setCategory()``.
+
+.. code-block:: php
+
+    <?php
+
+        use Sylius\Component\Shipping\Model\ShippingMethod;
+        use Sylius\Component\Shipping\Model\ShippingCategory;
+        use Sylius\Component\Shipping\Model\ShippingMethodInterface;
+
+        $shippingCategory = new ShippingCategory();
+        $shippingCategory->setName('Regular'); // Regular weight items
+
+
+        $shippingMethod = new ShippingMethod();
+        $shippingMethod->setCategory($shippingCategory); //default null, detach
+        $shippingMethod->getCategory(); // Output will be ShippingCategory object
+        $shippingMethod->setCategory(null);
+
+Setting Rule
+~~~~~~~~~~~~
+
+Every shipping method can have many rules, which defines additional requirements for shipment. If shipment do not
```

Comment:
`[...] which define its additional requirements. If a **Shipment** does not fulfill these requirements (e.g. a rule states that the expected quantity of shipment items should be 2, but the Shipment has only one **ShippingItem**), then the **ShippingMethod** having this rule cannot be used on this **Shipment**.`